cmd/sandbox-ce/cmd/config.go

```diff
@@ -0,0 +1,216 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/src-d/go-cli.v0"
+)
+
+const orgName = "smacker"
```

Comment:
it's `src-d` now

Reply:
fixed here https://github.com/src-d/superset-compose/pull/29/files#diff-28997035031024d6feab4265612f9909R19